app/src/main/java/com/santimattius/template/MainApplication.kt

```diff
@@ -15,6 +16,7 @@ class MainApplication : Application(), KoinStartup {
     override fun onKoinStartup(): KoinConfiguration {
         return KoinConfiguration {
             androidContext(this@MainApplication)
+            analyticsLogger()
```

Comment:
use `analytics(this@MainApplication)` , it will do analyticsLogger() +  KotzillaSDK.setup() directly on its own here. It's our one-line setup

Reply:
Thanks, I was looking at that function in the source code.